docs/book/advanced-guide/manage-your-secrets.md

```diff
@@ -213,7 +213,7 @@ register your secrets manager in the following way:
 
 ```shell
 zenml integration install aws
-zenml secrets-manager register AWS_SECRETS_MANAGER_NAME --flavor=aws
+zenml secrets-manager register AWS_SECRETS_MANAGER_NAME --flavor aws
```

Comment:
Is it really wrong to use = with --flavor ?

Reply:
No not wrong, but nowhere else really do we use the `=` for flavor assignment. It was just a smaller thing I noticed while I was in the document. The main change is the other one.

Reply:
I reverted this specific change now, @AlexejPenner, following your comment.